test/test_sparse.py

```diff
@@ -37,6 +37,10 @@ def outer(self, *args, **kwargs):
         inner(self, *args, **kwargs)
     return outer
 
+def _get_torch_cuda_version():
```

Comment:
This function is moved to the bottom of `common_cuda.py`. If you are gonna revert this, make sure handle that as well. I don't see it's reverted in the current diff. 

Or I think it's better to have that in `common_cuda.py` instead of `test_sparse.py`?

Reply:
There are some tests in `test_sparse` using `_get_torch_cuda_version`. The usage in `test_torch.test_inverse` can be reverted to the old `skipCUDAIfNoMagma`. Then, this `_get_torch_cuda_version` is only used in `test_sparse`. 

You may choose to have it in `test_sparse` or `common_cuda`.